### PR TITLE
use_shadow_variables

### PR DESCRIPTION
**Pull request by Feichtmeier**
Discussed with @madsrh 

Is this okay for you @clobrano ?

### CODE REVIEW

File: Communitheme/gtk-3.0/_colors.scss [hunk 1]
```diff
@@ -80,3 +80,7 @@ $hb_pathbar_bg: lighten($headerbar_bg_color, 5%);
 $hb_pathbar_bg_backdrop: mix(lighten($headerbar_bg_color, 5%), $headerbar_bg_color, 50%);
 $hb_pathbar_border_backdrop: darken($hb_pathbar_bg_backdrop, 2%);
 $hb_button_bg_hover: lighten($headerbar_bg_color, 10%);
+
+//box-shadows
+$window_decoration_shadow: 0 3px 9px 1px transparentize(black, 0.5);
```
Comment (clobrano) [inline Communitheme/gtk-3.0/_colors.scss]: I do not see the reason to create a variable for an expression which is used only once

Comment (Feichtmeier) [reply Communitheme/gtk-3.0/_colors.scss]: Oh it should beused twice. For the app notifications too. Pushing this later.

Shell notifications, gtk windows, gtk app notifications  all have the same shadow. Shell is ofc not in here ;0 but this way it is very clear that it is not some random box shadow but rather one specific shadow. 
Same for popovers and context menu

Comment (clobrano) [reply Communitheme/gtk-3.0/_colors.scss]: Right and I agree, but I still see no reason to use same variable names to link different projects ;)

Comment (Feichtmeier) [reply Communitheme/gtk-3.0/_colors.scss]: In the shell it has no name (yet)

Comment (Feichtmeier) [reply Communitheme/gtk-3.0/_colors.scss]: @clobrano is this okay now?

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/gtk-communitheme-pr490`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/gtk-communitheme-pr490` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

File: Communitheme/gtk-3.0/_common.scss [hunk 2]
```diff
@@ -4501,7 +4500,7 @@ decoration {
   // see bug #722563
   $_wm_border: transparentize(black, 0.77);
 
-  box-shadow: 0 3px 9px 1px transparentize(black, 0.5);
+  box-shadow: $window_decoration_shadow;
```
Comment (clobrano) [inline Communitheme/gtk-3.0/_common.scss]: So are we changing the window decoration shadow as well?

I am not sure about this, I was actually thinking about increasing windows shadow. Ambiance has bigger shadows, also other nice looking systems (OSX) have way bigger shadows. That would improve also the separation between backdropped windows when are overlapping (and not tiled)

**Comment by clobrano** [comment]
Sorry, by mistake, this changes "app notification", not window-decoration. Is this intended?

**Comment by Feichtmeier** [comment]
App notification shadow = shell notification shadow = window shadow
Popover shadow = context menu shadow
:) that's current master and that should be this pr if I make a mistake I check later

**Comment by Feichtmeier** [comment]
@clobrano you are confusing me!!! xD
It was correct in the initial commit!
So

shell notification shadow = window shadow
App notification shadow = Popover shadow = context menu shadow

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/gtk-communitheme-pr490`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/gtk-communitheme-pr490` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Comment by didrocks** [comment]
A new test snap version is available using: `snap refresh communitheme --channel=edge/gtk-communitheme-pr490`.
Further available updates will track that pull request then.

Switch back to stable or edge snap with `snap refresh communitheme --stable` or `snap refresh communitheme --edge` once you are done with it!
You may want to download as well `gtk-common-themes` snap from `edge/gtk-communitheme-pr490` channel to test your snaps with those changes.

Replace `refresh` with `install` if you haven't installed the snap yet.

**Comment by clobrano** [comment]
> It was correct in the initial commit!

Ook, so we are using now one variable used just once :smile:

**Comment by Feichtmeier** [comment]
@clobrano :man_shrugging: yeah okay, but if we are going to use it somewhere else we can use that variables. If you really think its useless let's forget about this. I am not emotionally tied to this PR! :dancer:

**Comment by clobrano** [comment]
I can live with that, just let's try to avoid adding unnecessary code in the future (more code = more bugs) ;)

**Pull request merged**